crypto/modes/asm/ghash-x86_64.pl

```diff
@@ -613,6 +620,7 @@ sub reduction_alg9 {	# 17/11 times faster than Intel version
 .type	gcm_gmult_clmul,\@abi-omnipotent
 .align	16
 gcm_gmult_clmul:
+	$endbr
```

Comment:
If this adds an opcode here, then it will be before the .cfi_startproc,
and therefore make the stack unwinder fail

Reply:
Thanks.  I pushed a fix.

Reply:
What I also do not understand, why you only add that here, there are many more
functions here, look for ".cfi_startproc", what is the rationale for this?

Reply:
Only functions which are called indirectly need to start with `endbr64`.   Since most of
functions aren't called indirectly, they don't need `endbr64`.